src/client.ts

```diff
@@ -335,7 +341,7 @@ class PixelClient {
       method: 'GET',
       headers: {
         'X-Adzerk-Sdk-Version': this._versionString,
-        'X-Kevel-ApiKey': additionalOpts?.apiKey,
+        'X-Kevel-ApiKey': additionalOpts?.apiKey ?? this._options?.apiKey,
```

Comment:
This was a comment from Ash to help the user experience. I'll roll it back to original functionality for now.

Reply:
Oh, I like the fallback from additionalOpts to this._options. I'm just afraid of sending `X-Kevel-ApiKey: null` rather than not sending the header at all if apikey is not set.

Reply:
It diverges against establshed behaviour, we could roll this, but we'd have to announce it.

Reply:
This wasn't actually the topic, but good suggestion as fetch actually still sends something even if set to undefined and null. Have updated.

Reply:
That was actually what I was trying to suggest, but Akira is a better communicator than me!

Reply:
Ahhhhhhh, I get it now. Sorry @vkurup that's clearly my misread.

Reply:
Heh, no worries! It's a 2 way street and both of ours have had heavy traffic on them recently